cg/services/order_validation_service/rules/case_sample/rules.py

```diff
@@ -329,6 +329,30 @@ def validate_subject_ids_different_from_case_names(
     return errors
 
 
+def validate_subject_sex_consistency(
+    order: OrderWithCases,
+    store: Store,
+) -> list[SexSubjectIdError]:
+    errors: list[SexSubjectIdError] = []
+
+    for case_index, case in order.enumerated_new_cases:
+        for sample_index, sample in case.enumerated_new_samples:
```

Comment:
Can we replace this with
```suggestion
    for case_index, sample_index, sample in order.enumerated_new_samples:
```?

Reply:
fixed